ui/src/pages/OBProxy/New/BasicConfig.tsx

```diff
@@ -25,42 +35,34 @@ export default function BasicConfig({ form }: BasicConfigProps) {
     >
       <Row gutter={[16, 32]}>
         <Col span={8}>
-          <Form.Item
-            label={intl.formatMessage({
-              id: 'src.pages.OBProxy.New.D41C48E1',
-              defaultMessage: '资源名称',
-            })}
-            rules={[
-              {
-                required: true,
-                message: intl.formatMessage({
-                  id: 'src.pages.OBProxy.New.A0CBFC04',
-                  defaultMessage: '请输入',
-                }),
-              },
-            ]}
-          >
-            <Input
-              placeholder={intl.formatMessage({
-                id: 'src.pages.OBProxy.New.E9A192AB',
-                defaultMessage: '请输入',
-              })}
-            />
-          </Form.Item>
+          <TooltipPretty title={'k8s中资源的名称'}>
+            <Form.Item
+              label="资源名称"
+              name={'name'}
+              rules={[
+                {
+                  required: true,
+                  message: '请输入k8s资源名称',
+                },
+                {
+                  pattern: /\D/,
```

Comment:
This regex pattern seems to match non-digit character. Is this correct?

Reply:
I think this is correct, the rules here are the same as when creating a cluster.